Makefile

```diff
@@ -0,0 +1,39 @@
+APPS = asset-manager content-store govuk-content-schemas multipage-frontend \
+	publishing-api router router-api rummager specialist-frontend \
+	specialist-publisher static travel-advice-publisher
+
+all: test
```

Comment:
If we do that, we can't take advantage of the parallel clone. I was hoping to make better use of the dependency features of `make` so it can detect when its not necessary to rebuild all the containers.